sale_order_priority/models/stock_picking.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models
+
+
+class Picking(models.Model):
+    _inherit = "stock.picking"
+
+    def action_assign(self):
+        res = super(Picking, self).action_assign()
```

Comment:
super().action_assign()

Reply:
pre-commit fails if I don't return the super

Reply:
the suggestion was not to delete the `res=`  + `return`. please, put this again. the suggestion was to delete the content inside super()